modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java

```diff
@@ -715,9 +715,8 @@ private String _getLocaleDisplayNames(Locale locale, Locale... locales) {
 	}
 
 	private String _getSaveAndContinueRedirect(
-			ActionRequest actionRequest, JournalArticle article,
-			String redirect)
-		throws Exception {
+		String actionName, ActionRequest actionRequest, JournalArticle article,
```

Comment:
ActionRequest/ActionResponse normally goes first place

Reply:
@AliciaGarciaGarcia, isn't it alphabetically ordered?

Reply:
@jkappler knows best

<img src="https://media4.giphy.com/media/SAW1EzSZCHrG0/giphy.gif"/>

Reply:
No, lets go with the recommendation of @AliciaGarciaGarcia. @NemethNorbert, can you change the order please (sorry, my bad for specifying it like this).

Reply:
I reordered it

Reply:
Thanks @NemethNorbert!